tests/docker/mounts.bats

```diff
@@ -265,11 +265,10 @@ function teardown() {
   local syscont=$(docker_run --rm --mount type=bind,source=${testDir},target=/var/lib/docker nestybox/alpine-docker-dbg:latest tail -f /dev/null)
 
   # This docker run is expected to pass but generate a warning (multiple containers can (but should not) share the same /var/lib/docker mount source)
-  run __docker run --runtime=sysbox-runc -d --rm --mount type=bind,source=${testDir},target=/var/lib/docker nestybox/alpine-docker-dbg:latest tail -f /dev/null
-  [ "$status" -eq 0 ]
-
+  local syscont_2=$(docker_run --rm --mount type=bind,source=${testDir},target=/var/lib/docker nestybox/alpine-docker-dbg:latest tail -f /dev/null)
```

Comment:
Why remove the [ "$status" -eq 0 ] check?

Reply:
As just discussed, there's no need for that.

Reply:
I see ... it's not needed because it's inside the "docker_run" bash function already ... disregard.